specification/quota/resource-manager/Microsoft.Quota/preview/2021-03-15-preview/examples/getComputeOneSkuUsages.json

```diff
@@ -0,0 +1,22 @@
+{
+  "parameters": {
+    "scope": "00000000-0000-0000-0000-000000000000/providers/Microsoft.Compute/locations/eastus",
```

Comment:
>00000000 [](http://example.com/codeflow?start=14&length=8)

This should include the subscriptions segment also

Reply:
I will update it.

Reply:
Updated it.